scripts/ultimate-upscale.py

```diff
@@ -532,7 +537,7 @@ def run(self, p, _, tile_size, mask_blur, padding, seams_fix_width, seams_fix_de
             p.height = math.ceil((init_img.height * custom_scale) / 64) * 64
 
         # Upscaling
-        upscaler = USDUpscaler(p, init_img, upscaler_index, save_upscaled_image, save_seams_fix_image, tile_size)
+        upscaler = USDUpscaler(p, init_img, upscaler_index, save_upscaled_image, save_seams_fix_image, tile_width, tile_heiht)
```

Comment:
typo: tile_heiht -> tile_height

Reply:
Yeah you're right, I'll do the corrections.